docs/_articles/en/codebuilder/cb-setup.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Set Up Code Builder
+lang: en
+---
+
+## Install the Code Builder Managed Package (Ss much detail is not necessary?)
+
+Salesforce Code Builder is a second-generation (2GP) managed package that you can install in any supported Salesforce org edition. Before you install, make sure you have enabled **Unlocked Packages and Second-Generation Managed Packages** in your Dev Hub.
+
+1. Click the install [link](https://login.salesforce.com/packaging/installPackage.apexp?p0=04t6g000008nwJuAAI)
```

Comment:
Does this generic install link work based on the My Domain changes? I'd have to ask Elaine Rose or try to parse her notifications about it.
In DevOps Center - we don't provide a link but say:
https://<your-domain>.lightning.force.com/packaging/installPackage.apexp?p0=04txxxxxxx

Reply:
Ah, you are getting them to log in first, and then the package installer displays after auth?